shopelectro/context.py

```diff
@@ -0,0 +1,368 @@
+"""
+Contains view context classes.
+
+Context is our own concept.
+It's part of the View abstract layer in MTV paradigm.
```

Comment:
MTV -> MVT

Reply:
@artemiy312 , seems it's called `MTV`. Like one popular TV-channel

> If you’re hungry for acronyms, you might say that Django is a “MTV” framework – that is, “model”, “template”, and “view.” That breakdown makes much more sense.

https://docs.djangoproject.com/en/2.1/faq/general/#django-appears-to-be-a-mvc-framework-but-you-call-the-controller-the-view-and-the-view-the-template-how-come-you-don-t-use-the-standard-names